modules/ROOT/pages/planning-and-tuning/query-tuning/indexes.adoc

```diff
@@ -187,6 +187,10 @@ However, other predicates are only used when it is known that the property is co
 
 This means that a `TEXT` index is not able to solve e.g. `a.prop = b.prop`, unless a type constraint also exists on the property.
 
+One can add xref:values-and-types/type-predicate.adoc[type predicate expressions] such as `IS {two-colons} STRING NOT NULL` to enforce the String type and possibly the property's existence.
+Note that `null IS {two-colons} STRING` is true and that indexes do not store entities with `null` properties.
+That is why, to be able to use the index, one has to use `IS {two-colons} STRING NOT NULL`.
+
```

Comment:
Could the "Note that `null IS {two-colons} STRING` is true" be clarified? Particularly the "is true" part?

```suggestion
As of Neo4j 5.15, it is possible to add xref:values-and-types/type-predicate.adoc[type predicate expressions], such as `IS {two-colons} STRING NOT NULL`, to enforce the `STRING` type and the property's existence.
Note that `null IS {two-colons} STRING` is true and that indexes do not store entities with `null` properties.
That is why, to be able to use the index, `IS {two-colons} STRING NOT NULL` must be used.
```